libfreerdp/core/gateway/http.c

```diff
@@ -572,6 +573,13 @@ static BOOL http_response_parse_header_field(HttpResponse* response, const char*
 		if (!response->ContentType)
 			return FALSE;
 	}
+	else if (_stricmp(name, "Transfer-Encoding") == 0)
+	{
+		response->TransferEncoding = value;
```

Comment:
It's rather suspicious that here we take an allocated value, and in `http_response_new` we put a static value. So either there's some lost memory, or we `free()` a static memory location.

Reply:
actually I wondered how value is allocated, but whereever it was allocated it will be freed there since response->TransferEncoding is only (second) a pointer to that string.

Anyway a better solution would be setting TransferEncoding to a enum, ideally to the same as the one used in rdg.c